openff/bespokefit/executor/services/qcgenerator/worker.py

```diff
@@ -75,10 +93,9 @@ def compute_torsion_drive(task_json: str) -> TorsionDriveResult:
             model=task.model, driver=DriverEnum.gradient
         ),
         optimization_spec=OptimizationSpecification(
-            procedure=task.optimization_spec.procedure,
+            procedure="geometric",
```

Comment:
```suggestion
            procedure=task.optimization_spec.program,
```
I think this is what is used further down?

Reply:
Yes that is better this should allow use to easily swap out the optimiser in future as well!